src/pages/account/resetPassword.tsx

```diff
@@ -1,19 +1,61 @@
 import styled from '@emotion/styled';
-import type { NextPage } from 'next/types';
+import type { GetServerSideProps, NextPage } from 'next/types';
+import * as api from 'api';
 import { ResetPasswordForm } from 'components/account';
 import { Seo } from 'components/common';
+import { PAGE_PATH } from 'constants/common';
+import { getQueryParam } from 'utils';
 
-const ResetPassword: NextPage = () => {
+interface PageProps {
+  email: string;
+  token: string;
+}
+
+const ResetPassword: NextPage<PageProps> = ({
+  email,
+  token,
+}: {
+  email: string;
+  token: string;
+}) => {
```

Comment:
p3; 
요렇게 하면 PageProps 따로 선언하지 않아도 됩니다!
```suggestion
const ResetPassword: NextPage<
  InferGetServerSidePropsType<typeof getServerSideProps>
> = ({ email, token }) => {
...
}

export const getServerSideProps = (async (context) => {
  ...
}) satisfies GetServerSideProps;
```

<br />

또는 PageProps를 적용하고자 한다면, 다음과 같이 해주시면 좋을 것 같아요!
그리고 타입 네이밍도 해당 페이지를 명시적으로 나타내는 네이밍으로 수정해주시면 좋을 것 같습니다!

```suggestion
const ResetPassword: NextPage<ResetPasswordPageProps> = ({ email, token }: ResetPasswordPageProps) => {
```

Reply:
`InferGetServerSidePropsType<typeof getServerSideProps>` 을 사용하면 타입을 명시적으로 적용하지 않아서 인지 그 값을 사용하는 부분에서 지정된 타입 사용을 요구한다면 타입스크립트 에러가 발생하여 타입을 직접 명시해주었습니다!

![스크린샷 2024-05-06 오후 5 29 01](https://github.com/a-daily-diary/ADD.FE/assets/23066745/2cfb496a-7fd1-4a1e-bb0e-0f2d0caeaa81)

Reply:
아! 그게 이 부분 때문일거에요!
저도 이 부분 좀 헷갈려요 헿

```typescript
// InferGetServerSidePropsType 사용 시 satisfies GetServerSideProps 사용
export const getServerSideProps = (async (context) => {
  ...
}) satisfies GetServerSideProps;

// InferGetServerSidePropsType 사용하지 않을 경우
export const getServerSideProps: GetServerSideProps = async (context) => {
...
} 
```

Reply:
앗 제가 `satisfies` 빼먹었군요..! 적용했습니다 감사해요☺️ 
[035b040](https://github.com/a-daily-diary/ADD.FE/pull/239/commits/035b040a92d410a429ff615335a6227014776dc8)